### ISSUE TITLE
Synchronise efforts with PyTado

### ISSUE THREAD

**Issue by wmalgadey** [erwindouna/python-tado#97]
Hi @erwindouna,

I am the "maintainer" of https://github.com/wmalgadey/PyTado. It seems, that you already spent quite a good amount of time into an alternative python library for Tado and Tado X. And as I saw here https://github.com/home-assistant/core/pull/129600/files#r1858759217 there seems to be the plan to replace PyTado in the homeassistant Tado integration.

Does it make sense to maintain two libraries in parallel?

I am asking, because in regard to the comments on https://github.com/home-assistant/core/pull/129600, I would totally agree, that the library should handle the Tado X abstraction, and I was just starting to get into this 😄 

From my perspective there are some "issues" with our libraries:

- mine is called PyTado, but because somebody else used this name on PyPi, I had to choose the name python-tado on pypi
- yours is called python-tado and tadoasync on pypi 😃 
- mine is not async, but fully integrated into the home assistant integration
- yours is nicely asnyc, but I think integrating it into home assistant would be a complete rework of the integration, doesn't it?

**Comment by erwindouna** [comment]
Hello @wmalgadey,

Nice hearing from you! For the greater good of the open source community I would indeed be very much in favor of synchronizing our efforts. Let me elaborate a bit why I started the initiative:

- There was a time your library was not maintained and slow paced. This could lead to issues in quick resolving if needed for about 11k+ installations of the Tado integration within Home Assistant. Basically the main reason.
- Perhaps I obtained a wrong e-mail address from you, yet I never got replies.
- Based on your readme, "_I’m not looking to collaborate on anything actually_", I made the assumption there was no interest in speeding up or change functionality for Home Assistant.
- It was not async. 😄 

Extra (nerdy) points why I was interested in starting my own:
- I want to experiment with a modern ecosystem, which can be found in the [pre-commit](https://github.com/erwindouna/python-tado/blob/main/.pre-commit-config.yaml) and the [GitHub Actions](https://github.com/erwindouna/python-tado/tree/main/.github).
- Having 100% code coverage and using Syrupy to achieve this.
- And few other unworthy points...

I am very pleased to see your picking up activity again and reaching out. I'd very encourage collaboration and joining efforts! I'd be willing to step up and put my effort within PyTado and work on topics we align on. 😄 
Let me try to e-mail address I got and see if we can have a better communication line. Looking forward to your response!
